mga/create-network-node.go

```diff
@@ -33,8 +34,16 @@ func CreateNetworkNodeFx(w http.ResponseWriter, r *http.Request) (string, interf
 		return "", "", err
 	}
 	// TODO: check node.Type in warrant.AllowedTypes
-	// TODO: check unique node.Code
 	// TODO: check unique companyCode for company
+	_, err = network.GetNetworkNodeByCode(inputNode.Code)
+	if !errors.Is(err, network.ErrNodeNotFound) {
+		// If err is *not* ErrNodeNotFound, options are:
+		// - an error raised hitting the db
+		// - the node code is already present
+		// In both cases we must abort
+		log.Printf("error checking node code: %s", err.Error())
+		return "", "", err
+	}
```

Comment:
See the comment below for specs. Consider extracting the whole flow into a wrapper function since the same change we'll be reflected here. It could be something like:
```golang
if err := wrapperFunction(inputNode.Code); err != nil {
    log.Println(err)
    return "", "", err
}
```